src/node.cpp

```diff
@@ -1020,6 +1047,15 @@ bool Node::setparent(Node* p)
     if (parent)
     {
         child_it = parent->children.insert(parent->children.end(), this);
+
+#ifdef ENABLE_SYNC
+        if (parent->type == FILENODE)
+        {
+            // copy syncable from child (old version) to parent (new version)
+            parent->setSyncable(isSyncable());
```

Comment:
For a down-only sync, if a file in the cloud had been un-syncable, but then it gets replaced with an updated file, I think we should then sync that file, rather than maintain the old version's status?

Reply:
Nah. Current spec is that the file does not get re-downloaded.

Reply:
does !mSyncable mean this side can't be synced to the other side, or that the other side can't update this node?  thanks

Reply:
`mSyncable` on `Node` indicates whether the remote node can be synced to the local tree.